package.json

```diff
@@ -135,7 +135,7 @@
   "bundlesize": [
     {
       "path": "./build/ddocs/medic/_attachments/main.js",
-      "maxSize": "950 kB"
```

Comment:
>  PASS  ./build/ddocs/medic/_attachments/main.js: 950.2KB < maxSize 955KB (gzip) 

Unfortunately, we were right below the `950` line before these changes and have just bumped it over...